File: app/src/main/java/com/rl/ff_face_detection_terload/ui/activity/FaceRecognizeActivity.java
package com.rl.ff_face_detection_terload.ui.activity;

import android.os.Bundle;
import android.view.TextureView;
import android.view.Window;
import android.view.WindowManager;

import androidx.appcompat.app.AppCompatActivity;

import com.rl.ff_face_detection_terload.R;
import com.rl.ff_face_detection_terload.faceRecognize.FaceRecognize;

public class FaceRecognizeActivity extends BaseActivity {

    private static final String TAG = "FaceRecognizeActivity";

    private FaceRecognize faceRecognize;

    @Override
    public int getLayoutResID() {
        return R.layout.activity_face_recognize;
    }

    @Override
    public void inits() {
        TextureView mTextureView = findViewById(R.id.texture_view);
        faceRecognize = new FaceRecognize();
        faceRecognize.onCreate(mTextureView, this);
    }

    @Override
    protected void onResume() {
        super.onResume();
        faceRecognize.onResume();
    }

    @Override
    protected void onPause() {
        faceRecognize.onPause();
        super.onPause();
    }
}

File: app/src/main/java/com/rl/ff_face_detection_terload/ui/activity/FaceRecognizeActivity.kt
package com.rl.ff_face_detection_terload.ui.activity

import android.graphics.Color
import android.view.TextureView
import android.view.View
import com.rl.ff_face_detection_terload.R
import com.rl.ff_face_detection_terload.faceRecognize.FaceRecognize

class FaceRecognizeActivity : BaseActivity() {
    private var faceRecognize: FaceRecognize? = null
    override fun getLayoutResID() = R.layout.activity_face_recognize

    override fun onWindowFocusChanged(hasFocus: Boolean) {
        super.onWindowFocusChanged(hasFocus)
        window.apply {
            decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN or View.SYSTEM_UI_FLAG_LAYOUT_STABLE or View.SYSTEM_UI_FLAG_HIDE_NAVIGATION
            statusBarColor = Color.TRANSPARENT
        }
    }


    override fun inits() {
        val mTextureView = findViewById<TextureView>(R.id.texture_view)
        faceRecognize = FaceRecognize()
        faceRecognize!!.onCreate(mTextureView, this)
    }

    override fun onResume() {
        super.onResume()
        faceRecognize!!.onResume()
    }

    override fun onPause() {
        faceRecognize!!.onPause()
        super.onPause()
    }

    companion object {
        private const val TAG = "FaceRecognizeActivity"
    }
}